puppet/modules/site_module/files/20-cgroupfs.conf

```diff
@@ -0,0 +1,16 @@
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd-current \
+          --add-runtime docker-runc=/usr/libexec/docker/docker-runc-current \
+          --default-runtime=docker-runc \
+          --exec-opt native.cgroupdriver=cgroupfs \
+          --userland-proxy-path=/usr/libexec/docker/docker-proxy-current \
+          --init-path=/usr/libexec/docker/docker-init-current \
+          --seccomp-profile=/etc/docker/seccomp.json \
```

Comment:
Have you tried if a second `--exec-opt native.cgroupdriver=cgroupfs` in OPTIONS is able to override the first one @dippynark

Reply:
yeah, it didn't work